onnxruntime/core/session/inference_session.h

```diff
@@ -132,6 +139,8 @@ class InferenceSession {
   */
   common::Status LoadCustomOps(const std::vector<std::string>& dso_list);
 
+  common::Status AddCustomOpDomains(const std::vector<OrtCustomOpDomain*>& ops);
```

Comment:
This API should be AddCusotmOps<list of ops>. The domain thing will be added implicitly when adding custom op internally.